Findexternpro.cmake

```diff
@@ -84,13 +91,13 @@ find_path(externpro_DIR
     C:/src/externpro/_bld/externpro_${externpro_SIG}
     ~/src/externpro/_bld/externpro_${externpro_SIG}
     # environment variable
-    "$ENV{externpro_DIR}/externpro ${externpro_SIG}"
-    "$ENV{externpro_DIR}/externpro-${externpro_SIG}-${CMAKE_SYSTEM_NAME}"
+    "$ENV{extern_DIR}/externpro ${externpro_SIG}"
+    "$ENV{extern_DIR}/externpro-${externpro_SIG}-${CMAKE_SYSTEM_NAME}"
     # installed versions
     "$ENV{ProgramW6432}/externpro ${externpro_SIG}"
     "$ENV{${PFX86}}/externpro ${externpro_SIG}"
-    "~/externpro/externpro-${externpro_SIG}-${CMAKE_SYSTEM_NAME}"
-    "/opt/externpro/externpro-${externpro_SIG}-${CMAKE_SYSTEM_NAME}"
+    "~/extern/externpro-${externpro_SIG}-${CMAKE_SYSTEM_NAME}"
+    "/opt/extern/externpro-${externpro_SIG}-${CMAKE_SYSTEM_NAME}"
```

Comment:
Why these changes? And those on lines 94 and 95?

Reply:
@brobeson  this file is a copy from externpro.

Reply:
this is now the default install location on unix systems: `/opt/extern` and `~/extern` because there are other projects (for SkyLynx development) that are using or modeled after externpro and we decided that a clean start and a more general location was warranted